pySPT/preAnalysis/diffLimit_noGUI.py

```diff
@@ -0,0 +1,267 @@
+"""
+@author: Johanna Rahm
+Research group Heilemann
+Institute for Physical and Theoretical Chemistry, Goethe University Frankfurt a.M.
+
+Determine the minimal distance localizations can be told apart based on nearest neighbor analysis.
```

Comment:
adjust docstring description (add your name under authors, add sentence that file has deactivated plot output and is used for batch processing)